pupil_src/shared_modules/video_capture/utils.py

```diff
@@ -173,8 +173,14 @@ def __init__(self, path: str) -> None:
         self.ts = None
 
     def check_container(self):
-        cont = av.open(self.path)
-        frame = next(cont.decode(video=0))
+        try:
+            cont = av.open(self.path)
+            n = cont.decode(video=0)
+            _ = next(n)
+        except av.AVError:
+            return
+        else:
+            return av.open(self.path)
```

Comment:
Why do you call `open()` a second time? Isn't it faster to seek back to the zero'th frame and return `cont`?